tests/periph_gpio/main.c

```diff
@@ -188,7 +188,7 @@ static int enable_int(int argc, char **argv)
 }
 #endif
 
-static int read(int argc, char **argv)
+static int cmd_read(int argc, char **argv)
```

Comment:
In this file the naming is not really consistent with the other commands. But I have no better idea.

Reply:
Yea that's why I renamed the other commands too.